cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -792,29 +792,111 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             ctx.emit(Inst::Ud2 { trap_info })
         }
 
+        Opcode::F64const => {
+            // TODO use xorpd for 0
```

Comment:
Whatever is chosen, make sure it's an officially-recognised dependency-breaking insn.